COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md

```diff
@@ -820,14 +702,12 @@ If you do not see a version number after you submit the command, you do not have
 [`https://www.java.com/en/download/help/uninstall_java.xml` ](https://www.java.com/en/download/help/uninstall_java.xml)
 
 
-3. Once Java is uninstalled from your workstation, you can click the Java JDK 8 download link below and follow the installation instructions for your specific OS.
+3. Once Java is uninstalled from your workstation, you can click the Java JDK 17 download link below and follow the installation instructions for your specific OS.
 
 [`https://www.oracle.com/java/technologies/javase-jdk8-downloads.html` ](https://www.oracle.com/java/technologies/javase-jdk8-downloads.html)
```

Comment:
Hmm, links seems to be for JDK 8? Does Zowe or OMP have a recommended Java runtime (e.g., Oracle or OpenJDK or IBM Semeru or Eclipse Temurin or something else)?

Reply:
They generally recommend installing JDK and now with the new zowe version it says you should use Java version 17 or above. My bad I missed updating the link.